main.go

```diff
@@ -15,6 +16,9 @@ import (
 
 func main() {
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
+	if info, ok := debug.ReadBuildInfo(); !ok && info == nil {
+		fmt.Fprint(os.Stderr, "Warning: dnscontrol was built without Go modules. See https://github.com/StackExchange/dnscontrol#from-source for more information on how to build dnscontrol correctly.\n\n")
```

Comment:
@tlimoncelli, what do you think of this message?

Reply:
:+1: